sync/syncer.go

```diff
@@ -244,18 +253,21 @@ func (s *Syncer) Status() string {
 
 func (s *Syncer) Start() {
 	if s.startLock.TryLock() {
+		s.Info("start syncer")
 		go s.run()
 		s.forceSync <- true
 		return
 	}
+	s.Info("syncer already started")
 }
 
 //fires a sync every sm.SyncInterval or on force space from outside
 func (s *Syncer) run() {
+	s.Info("Start running")
```

Comment:
Please add context: what is starting to run?

Reply:
this is the main sync loop i don't think it warrants context